docs/ja-jp/3.6-インフラスケジュール設定.md

```diff
@@ -0,0 +1,108 @@
+            },
+            "rooms": {              // 部屋情報，必須
+                                    // 引数 "control" / "manufacture" / "trading" / "power" / "meeting" / "hire" / "dormitory"
+                                    // 1つもないということは、その施設ではシフト変更にデフォルトのアルゴリズムが使用されていることを意味します。
+                                    // 部屋のシフトを変更しない場合は、skip を使用するか、設定 - 基地設定 で該当施設のチェックを外すだけです。
+                "control": [
+                    {
+                        "operators": [
+                            "シー",   // OCRを使用してこれを行うには、対応するクライアント言語のオペレーター名を渡す必要があります。
+                            "リィン",   // JP版なら日本語で入力しろってことですね。
+                            "ケルシー",
+                            "アーミヤ",
+                            "ムリナール"
+                        ]
+                    }
+                ],
+                "manufacture": [
+                    {
+                        "operators": [
+                            "フェン",
+                            "シーン",
+                            "クルース"
+                        ],
+                        "sort": false,  // ソートするかどうか（上の オペレーター の順番で）、オプション、デフォルトはfalse
+                    },
+                    {
+                        "skip": true    // 現在の施設をスキップするかどうか（配列番号に対応），オプション，デフォルトは false
+                                        // true の場合、他のすべてのフィールドを空にすることができる。 オペレーター交代の操作だけが省略され、ドローンの使用や手がかりの交換など、他のことは今まで通り行われます。
+                    },
+                    {
+                        "operators": [
+                            "Castle-3"
+                        ],
+                        "autofill": true,   // 残った位置を自動的に埋めるためにオリジナルのアルゴリズムを使用、オプション、デフォルトは false
+                                        // operators が空の場合、部屋はオリジナルのアルゴリズムで全体的にスケジュールされる
+                                        // operators が空でない場合、施設全体の効率ではなく、単一のオペレータの効率のみで考慮されます。
+                                        // 後でカスタムシステムと競合する可能性があることに注意してください。たとえば、後で必要な関数がここで使用される場合、注意するか、autofill を最後に置いてください
+                        "product": "Battle Record"  // 現在の製造資材、オプション
+                                                    // 現在の設備がジョブで設定された製品と一致しないことが確認された場合、インターフェースは赤いメッセージでポップアップ表示され、後でより便利になる可能性があります
+                                                    // 使用可能引数： "Battle Record" | "Pure Gold" |  "Dualchip" | "Originium Shard" | "LMD" | "Orundum"
+                    },
+                    {
+                        "operators": [
+                            "ドロシー"
+                        ],
+                        "candidates": [ // サブオペレーター，オプション。埋まるまで、誰でも利用可能です
+                                        // autofill=true との互換性はありません。空白でない場合、autofill は false である必要があります。
+                            "アステジーニ",
+                            "フィリオプシス",
+                            "サイレンス"
+                        ]
+                    }
+                ],
+                "meeting": [
+                    {
+                        "autofill": true // この施設はすべて autofill
```

Comment:
The Chinese here is "This room is autofill", I used DeepL to tell me that "All of these rooms are autofill" here in Japanese, this may not be true. @Tastror

Reply:
すべて（全部）是修饰 autofill 的，不是修饰 施設 的。施設 前面是 この（这个）。
所以不要用 DeepL 翻日语.jpg

Reply:
Change Text.
Change from whole sentences to single words.

//
どちらにせよ応接室は現状固定施設のため、”応接室すべて”でも問題はないようには思うのですが・・・
今後のアップデートで増えたりする可能性はあるので上記の方が良いかもしれませんね。

Either way, I don't see the problem with "この施設はすべて autofill" since reception rooms are currently fixed facilities...
However, it is possible that the number may increase in future updates, so the above may be a better choice.

Reply:
気にしないで、これは DeepL 翻訳の問題です

Reply:
Users may refer here and then use `autofill` for other facilities that have multiple rooms. No problem with the new commit.